src/components/views/settings/tabs/room/AdvancedRoomSettingsTab.tsx

```diff
@@ -169,12 +164,6 @@ export default class AdvancedRoomSettingsTab extends React.Component<IProps, ISt
                     { oldRoomLink }
                     { roomUpgradeButton }
                 </div>
-                <div className='mx_SettingsTab_section mx_SettingsTab_subsectionText'>
-                    <span className='mx_SettingsTab_subheading'>{ _t("Developer options") }</span>
-                    <AccessibleButton onClick={this.openDevtools} kind='primary'>
-                        { _t("Open Devtools") }
-                    </AccessibleButton>
-                </div>
```

Comment:
we should maintain this button, though behind a developer flag. It's for when you can't type `/devtools` in the composer because it's disabled (tombstone, announcement rooms, etc).

Reply:
There's now a button in the quick settings panel

![image](https://user-images.githubusercontent.com/2403652/159429127-df747a03-13aa-4f51-af3c-ea3613bdd2a1.png)

Reply:
![image](https://user-images.githubusercontent.com/2403652/159761969-5c86acda-ea99-4b95-8d21-206d7f8a4ea3.png)

Added another button